### ISSUE TITLE
Task 'migrate' is not in your gulpfile

### ISSUE THREAD

**Issue by felipebueno** [cdubz/babybuddy#1]
Hello!

First of all, kudos for the nice project :)

The error bellow was thrown for the ```gulp migrate``` command. I was following the installation instructions on README.

```
(babybuddy-gnv08_rE) ➜  babybuddy git:(master) gulp migrate
[11:55:46] Using gulpfile ~/Devel/tools/babybuddy/gulpfile.js
[11:55:46] Task 'migrate' is not in your gulpfile
[11:55:46] Please check the documentation for proper gulpfile formatting
```

**Comment by cdubz** [comment]
Argh! That is a very unfortunate oversight. Thanks for reporting. I'll sort it out this evening.

In the meantime, you can can do ``pipenv run python manage.py migrate`` and then ``gulp``.

**Comment by cdubz** [comment]
Lunch time fix incoming (28c2238). Thanks again for reporting!
